cypress/integration/registrationFlows/emailRegistration.spec.js

```diff
@@ -0,0 +1,27 @@
+describe('Sign up with email', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.visit('/enter?state=new-user');
+  });
+
+  // Regression test for #11099
+  it('should preserve the form data on unsuccesful submissions', () => {
+    cy.get('a').contains('Sign up with Email').click();
+
+    cy.get('input[type="file"]').attachFile('images/admin-image.png');
+    cy.getInputByLabel('Name').type('Sloan');
```

Comment:
I mentioned it in the other file, but we can just use e.g. `cy.findByLabelText('Name')` for these ones 🙂